src/main/java/lotto/domain/entity/Lottos.java

```diff
@@ -0,0 +1,39 @@
+package lotto.domain.entity;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Lottos {
+    private final List<Lotto> lottos;
+
+    private Lottos(List<Lotto> lottos) {
+        this.lottos = lottos;
+    }
+
+    public static Lottos create(List<Lotto> lottos) {
+        return new Lottos(lottos);
+    }
+
+    public static Lottos createAutomatic(ThousandUnitMoney purchaseAmount, int oneLottoPrice) {
+        int count = purchaseAmount.quotient(oneLottoPrice);
```

Comment:
만약 ThousandUnitMoney에서 amount가  int형에서 long형으로 바꿔야한다는 요구사항이 추가된다면 어떻게할까요? 

현재 Lottos.createAutomatic()에서 현재 2번째 인수로 int형을 받습니다. 하지만 long형을 받는다면 해당 메서드를 사용하는 모든 클래스는 변경에 영향을 받게 됩니다.

-> LottoPrice를 똑같이 ThousandUnitMoney로 변경해보는 것은 어떨까요?

```suggestion
    public static Lottos createAutomatic(ThousandUnitMoney purchaseAmount, ThousandUnitMoney oneLottoPrice) {
        int count = purchaseAmount.quotient(oneLottoPrice);
```